README.md

```diff
@@ -1,7 +1,22 @@
 # **Observability Remote Write Proxy**
 
-### A proxy to authenticate remote write requests before they are forwarded to Observatorium. 
+### A proxy to accept unauthenticated remote write requests and forward them to Observatorium. Before forwarding the request, the proxy authenticates with the provided oidc credentials and adds the obtained token to the request. The proxy is intended to be used by customers who want to send data to Observatorium but don't want to store Observatorium credentials on their clusters.
```

Comment:
Let's remove the word `unauthenticated` here, because there might (and in fact, are) still other auth tokens on the request.